frontend/src/metabase/parameters/components/widgets/StringInputWidget/StringInputWidget.unit.spec.tsx

```diff
@@ -69,10 +69,12 @@ describe("StringInputWidget", () => {
       );
 
       const textbox = screen.getByRole("textbox");
-      await userEvent.type(textbox, "{backspace}{backspace}{backspace}bar");
+
+      await userEvent.type(textbox, "bar");
+
       const button = screen.getByRole("button", { name: "Update filter" });
       await userEvent.click(button);
-      expect(mockSetValue).toHaveBeenCalledWith(["bar"]);
+      expect(mockSetValue).toHaveBeenCalledWith(["foobar"]);
```

Comment:
This was by far the most frustrating test to migrate. Originally, the value was cleared by using backspaces, but once the input had an empty value and we started typing, the single character from before it was empty would be appended to the setValue call. So the call would always be made with ["fbar"]. I wasn't able to reproduce this in the app, and after spending over an hour on this I decided to change the test. Happy to take a look with someone else if they're interested in pairing.